crates/humanode-peer/src/chain_spec.rs

```diff
@@ -80,6 +83,16 @@ fn dev_robonode_public_key(default: &'static [u8]) -> Result<robonode::PublicKey
     .map_err(|err| format!("unable to parse robonode public key: {err:?}"))
 }
 
+/// A helper function to construct non system evm genesis account with predefined balance.
```

Comment:
```suggestion
/// A helper function to construct an EVM genesis account with a predefined balance.
```

We don't have a notion of non-system really, cause there's either a native system or EVM system...